src/mintpy/save_hdfeos5.py

```diff
@@ -250,7 +268,11 @@ def get_output_filename(metadata, suffix=None, update_mode=False, subset_mode=Fa
 
         SUB = f'_{lat0Str}_{lat1Str}_{lon0Str}_{lon1Str}'
         fbase, fext = os.path.splitext(outName)
-        outName = f'{fbase}{SUB}{fext}'
+
+        if suffix:
+            outName = fbase.removesuffix('_' + suffix) + SUB + '_' + suffix + fext
+        else:
+            outName = fbase + SUB + fext
```

Comment:
Given the `suffix` is used [here](https://github.com/insarlab/MintPy/blob/c7400f97ffd370b0140906cbf3925ead6de10fc1/src/mintpy/save_hdfeos5.py#L230-L233), I don't see the necessity of this change.

Reply:
Are you referring to removesuffix? Well, it was needed. Otherwise I would not have added it. fbase seems to have occasionally an `_`.
You could make a mote to the code that this might be unnecessary and I can revisit next time I deal with this.

Reply:
Please provide enough detailed examples, so that I could test this change. Otherwise, we can not accept the PR here.